### ISSUE TITLE
Does Genoa need a new firmware update to enable SNP ?

### ISSUE THREAD

**Issue by codecrypt92** [AMDESE/AMDSEV#148]
I am using 0x19h for Genoa but the firmware doesn't get updated. Am I missing some step here ? I followed the upgrade firmware steps as outlined and it doesn't work on Genoa. It works on Milan though. @tlendacky @codomania 

When I do a dmesg, I don't see the firmware update successful message.

**Comment by tlendacky** [comment]
The firmware is family/model specific. There isn't a published downloadable version of the SEV firmware for Genoa, yet.

**Comment by codecrypt92** [comment]
Thanks for the update. Any pointers to when it might be released ?

**Comment by codecrypt92** [comment]
Also, just to make sure I fully comprehend, I cannot use SNP on the new platform till the new firmware is released ? Is there a timeframe to this ? Because, I don't remember seeing this anywhere.

**Comment by tlendacky** [comment]
> Thanks for the update. Any pointers to when it might be released ?

I don't have any information on firmware release dates.


> Also, just to make sure I fully comprehend, I cannot use SNP on the new platform till the new firmware is released ? Is there a timeframe to this ? Because, I don't think I have read about SNP differing on enabling timelines on Genoa anywhere.

The 1000 release of AGESA should have the proper level of SEV firmware, so try upgrading your BIOS to the latest version available for your system to see if that takes care of it.

**Comment by codecrypt92** [comment]
Yes, I am already using the latest BIOS for Genoa, which has all the required knobs on the BIOS (enabling RMP & SNP), but I guess it is the SEV firmware for Genoa which is preventing from enabling SNP. I can only enable SEV-ES.
